docs/DEVGUIDE.md

```diff
@@ -12,6 +12,7 @@ layout:
     ├── build          # Contains build-related scripts and subdirectories containing Dockerfiles
     ├── cmd            # Contains "main" Go packages for each service catalog component binary
     ├── contrib        # Contains all non-essential source
+    │   └── hack       # Non-build related scripts
```

Comment:
oh, I see, it moved inside. do we need to specify it's there? there's a lot of stuff inside contrib.